_data/fa-owners.yml

```diff
@@ -0,0 +1,353 @@
+'API definition':
+  - area: 'API definition'
+    label: api-definition
+    owner:
+    - sq-lb-epitome
+    repos:
+    - loopback-api-definition
+    - loopback-component-explorer
+    - loopback-swagger
+
+'Boot':
+  - area: Boot
+    label: boot
+    owner:
+    - sq-lb-asteroid
+    repos:
+    - loopback-boot
+    - loopback-example-app-logic
+    smes:
+    - davidcheung
```

Comment:
I'm also siding with @bajtos, is it not possible to render the link text to show up as @user? So instead of "loay" in your pic, it shows "@loay" instead?